examples/apps/data-object-grid/src/container.ts

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ModelContainerRuntimeFactory } from "@fluid-example/example-utils";
+import { IContainer } from "@fluidframework/container-definitions";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+
+import { DataObjectGrid, IDataObjectGrid } from "./dataObjectGrid";
+
+export interface IDataObjectGridAppModel {
```

Comment:
Again, not sure how much we expect folks to be looking at this, but it might be worth adding docs to at least the exported members here.

Reply:
Done in latest